crates/esthri/src/aws_sdk.rs

```diff
@@ -129,10 +146,28 @@ pub struct GetObjectResponse {
     pub size: i64,
     pub part: i64,
 }
+/// The AWS SDK removed future stream implementation of a Byte Stream Wrapper,
```

Comment:
Kind of odd given that ByteStream has both `poll_next` and `size_hint`, wonder what the reasoning is behind that